src/chart/generator/axis.h

```diff
@@ -123,8 +134,7 @@ struct DimensionAxis
 	};
 	using Values = std::multimap<Data::MultiDim::SliceIndex, Item>;
 
-	Math::FuzzyBool enabled{false};
-	::Anim::String title;
+	bool enabled{false};
```

Comment:
Why this shouldn't be an interpolatable bool?

Reply:
Currently, the variable is not used as interpolated. The enabled is set on interpolation manually, and not interpolated.
https://github.com/vizzuhq/vizzu-lib/blob/main/src/chart/generator/axis.cpp#L124